tests/ui/drop/lint-tail-expr-drop-order.rs

```diff
@@ -2,7 +2,6 @@
 // This lint is to capture potential change in program semantics
 // due to implementation of RFC 3606 <https://github.com/rust-lang/rfcs/pull/3606>
 //@ edition: 2021
-//@ build-fail
```

Comment:
A `build-fail` became a `check-fail`, which is evidence enough of this being a sufficient fix.